cpu/mips_pic32_common/Makefile.include

```diff
@@ -1,3 +1,8 @@
 include $(RIOTCPU)/mips32r2_common/Makefile.include
 
+CFLAGS += -DCPU_FAM_$(call uppercase_and_underscore,$(CPU_FAM))
+
+# mips32 needs periph_timer for its gettimeofday() implementation
+USEMODULE += periph_timer
```

Comment:
This should be moved to `cpu/mips_pic32_common/Makefile.dep`

Reply:
Done